### ISSUE TITLE
Cannot focus on Start Over/Run Code buttons with keyboard - accessibility issue

### ISSUE THREAD

**Issue by murphyqm** [r-wasm/quarto-live#104]
This is such a fantastic tool, particularly for teaching - thanks for all the time and effort involved in this!

Only a small change needed to make this library/framework more accessible: since semantic html isn't used for the "Start Over" and "Run Code" buttons (they are <span> elements), they currently aren't focusable with keyboard, making it inaccessible/inoperable by keyboard navigation (can make screen-reader use difficult too!).

I've been fixing my pages by manually going in to the html that Quarto renders, and adding in `tabIndex=0` to the <span> elements: this makes these buttons focusable and allows you to use the enter key to activate the buttons! 

The lines of code I edit:

```html
<span class="btn-label-exercise-editor" tabindex="0">Start Over</span>

<span class="btn-label-exercise-editor" tabindex="0">Run Code</span>
```

Ideally, it should use a button element as opposed to spans/divs, but at least the above fix makes it operable!

I will have a go at fixing this but wanted to share the fix above in the meantime!
